index.css

```diff
@@ -12,10 +12,7 @@ body {
   box-shadow: 1px 1px;
   position: absolute;
   margin: 0;
-  font-family: 'Poppins' sans-serif;
-  
-  
-  
+  font-family: Poppins, sans-serif;
 }
 
```

Comment:
- [ ] [REQUIRED] Good jobs so far! Please adjust the position between the `sections` and give the correct `shape` to the section `border` because they are stuck together, and remove the black `line` on the right side of the web page🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/228014246-c8c7dc79-36c5-4c98-b53d-cd5cebc8b399.png)